### ISSUE TITLE
Dishonoured 2 menu freeze on DXVK 1.3.1

### ISSUE THREAD

**Issue by regulator-g** [doitsujin/dxvk#1148]
Symptoms

Using Proton 4.11/DXVK 1.3.1 
I launch Dishonoured 2.. It works for about a minute then quicksave/quickload stops working, I press ESC for the menu, it starts bringing up the menu then freezes. I can alt+tab and quit the game.

Additionally in-game when I look around I see shadows that appear then disappear as I move the mouse.

e.g.
https://imgur.com/a/o4FtkPv

No obvious messages in console

-vsync off
-fps_max 60
-triple buffering off

Proton 4.2/DXVK 1.2.1  does not have these issues

"dropping in the DXVK 1.2.1 build from Proton 4.2 into 4.11 and found that the Escape menu now works"

Compatibility Report

    Name of the game with compatibility issues: Dishonoured 2
    Steam AppID of the game: 403640

System Information

    GPU: RX 570
    Driver/LLVM version: Mesa 19.2 ACO
    Kernel version: 5.0.0-21
    Proton version: 4.11

Proton Log:
https://github.com/ValveSoftware/Proton/files/3459299/steam-403640.log

DX logs
[Dishonored2_d3d11.log](https://github.com/doitsujin/dxvk/files/3463298/Dishonored2_d3d11.log)

[Dishonored2_dxgi.log](https://github.com/doitsujin/dxvk/files/3463299/Dishonored2_dxgi.log)

Apitrace:
(is there not a PPA for apitrace?)

**Comment by Kreyren** [comment]
Referencing https://github.com/ValveSoftware/Proton/issues/823#issuecomment-517513954

**Comment by doitsujin** [comment]
Dishonored 2 is working fine on my end. Please record an apitrace or bisect the issue.

**Comment by Oschowa** [comment]
I can reproduce the hang and bisecting produced
```
commit 1cc531eaf8002cef3e9707cdc3398f45909e38d2
Author: Philip Rebohle <philip.rebohle@tu-dortmund.de>
Date:   Fri Jul 5 10:25:06 2019 +0200

    [d3d11] Return error when mapping an already mapped image
    
    Final Fantasy XIV does this. Should avoid some unnecessary work.

 src/d3d11/d3d11_context_imm.cpp | 3 +++
 1 file changed, 3 insertions(+)
```
as the first bad commit. Reverting it on master fixes the issue.

**Comment by jaubin** [comment]
Same here. Thanks for the fix ;-)

**Comment by regulator-g** [comment]
Hey guys saw 1.3.2 is out with the Dishonoured 2 fix, thanks heaps that was fast! soon enough proton will get updated too

**Comment by trialism** [comment]
I'm using proton 4.11.7 and the menu freeze is an issue again. The game runs fine but when I try to do something like delete a save or load one in the middle of the session the app hangs until I kill it.

**Comment by doitsujin** [comment]
I can't reproduce that. Both the menu and quicksave/quickload are working fine for me (although I'm on DXV 1.4.4, maybe try that)

**Comment by trialism** [comment]
@doitsujin There is no issue with quick save/load until I play for 5 minutes and I try to load stuff. I installed 1.4.4 for the proton version with protontricks and I also tried it with lutris/dxvk-1.4.4. With both installations, the game made the system and itself hang permanently after I played for 3-4 minutes and tried to load a save. The game also froze with regular gameplay once - when I tried to play for 10 minutes. The game also hangs when I switch virtual worskpaces or when I alt-tab away. The lutris version also had terrible fps(1-5) but that issue could be the missing shader cache.
Previously, dishonored 2 was only choppy when I entered new areas and there were times when the yes/no popup(also the current culprit) made the game really slow(1-2fps) and I needed to restart it.
